transposer/features/org.polarsys.kitalpha.transposer.feature/feature.xml

```diff
@@ -173,4 +171,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```

Comment:
Adding it in the root feature of transposer will add an hard dependencies to coevolution which is not embed by default into capella, preventing using transposer without diffmerge and diffmerge-coevolution.

Can you move it to the sample feature? (transposer/examples/features) as this plugin is still in incubation.

Reply:
hmm, the transposer/examples/features/org.polarsys.kitalpha.transposer.examples.feature is a bit messy and embed some weird stuff, can you create a new feature to handle this plugin. The plugin will be used intependently to the other samples